Functions.Templates/Templates/SqlOutputBinding-JavaScript/index.js

```diff
@@ -0,0 +1,15 @@
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    const testRow = {
+        "column1": column1Value,
```

Comment:
not sure having the template trying to insert a null value is a good starting point.  can we set this var ahead of line 4 to ... something? the number 12 etc

Reply:
I can totally get rid of that and have the testRow directly assigned to the results?

Reply:
I don't see why we have our samples have these hardcoded values anyways. That isn't generally what people are going to be doing I would image - you should just create the row object from the param values passed in.